spikeinterface/core/numpyextractors.py

```diff
@@ -115,6 +115,9 @@ def from_times_labels(times_list, labels_list, sampling_frequency):
             An array of spike times (in frames).
         labels_list: list of array (or array)
             An array of spike labels corresponding to the given times.
+        sorting_key: python function used to reorder the labels when creating
+            the new array. By default, they are sorted with np.sort, but more 
+            evolved sorting schemes could be needed
```

Comment:
This is a bit weird!
I would prefer to expose `unit_ids=None` directly
If None : then np.unique
if not None: then unit_ids in the given odrer

Reply:
ok, I could do that. I just want a way to have the unit_ids in a control order, in order to match my ground truths benchmarks